conf/pf.conf.defaults

```diff
@@ -1290,6 +1290,11 @@ acct_maintenance_interval=60s
 #
 # Interval at which Packetfence runs the polling enforcement on the provisioners
 provisioning_compliance_poll_interval=300s
+#
+# maintenance.online_node_state_update_interval
+#
+# Interval at which Packetfence updates the online node state
+online_node_state_update_interval=60s
```

Comment:
Is this too low for a high volume deployment?

We must absolutely avoid shooting ourselves in the foot.